docs/robotframework.rst

```diff
@@ -361,6 +361,8 @@ The **PageObjects** library provides the following keywords:
 * Go To Page Object
 * Load Page Object
 * Log Page Object Keywords
+* Wait For Dialog
```

Comment:
```suggestion
* Wait For Modal
```

Reply:
Ugh. Thanks for catching the robotfamework.rst problems. I grepped .robot and .py files but forgot the documentation files.